lib/ansible/modules/windows/win_scheduled_task.py

```diff
@@ -193,6 +193,14 @@
         - The interval of weeks to run on, e.g. C(1) means every week while
           C(2) means every other week.
         - Optional when C(type=weekly).
+      repetition:
+        description:
```

Comment:
I would have 1 sentence saying what the repetition field does at the beginning and another to show what all the valid keys are. Then from there you can have a sentence on each key, the only part I would change with those sentences is to enclose the key with `C()` and not put a newline, e.g.

```
repetition:
  description:
  - Allows you to define the repetition action of the trigger (you can jazzify it up if you want :).
  - It takes in the following keys, C(duration), C(interval), C(stop_at_duration_end)
  - C(duration) is how long the pattern is repeated and is written in the ISO 8601 Duration format C(P[n]Y[n]M[n]DT[n]H[n]M[n]S).
  - C(interval) is the amount of time...
  - C(stop_at_duration_end) is a boolean value that indicates ...
```